hugo.py

```diff
@@ -29,20 +29,33 @@
 
 ELASTICSEARCH_ENDPOINT = os.getenv("ELASTICSEARCH_ENDPOINT", "http://localhost:9200/")
 REPOSITORY_BRANCH = os.getenv("REPOSITORY_BRANCH", "main")
-REPOSITORY_SUBFOLDER = os.getenv("REPOSITORY_SUBFOLDER", "src/content")
-APIDOCS_BASE_URI = os.getenv("APIDOCS_BASE_URI", "https://docs.giantswarm.io/api/")
-APIDOCS_BASE_PATH = os.getenv("APIDOCS_BASE_PATH", "/api/")
-API_SPEC_FILES = os.getenv("API_SPEC_FILES", "yaml/spec.yaml,yaml/definitions.yaml,yaml/parameters.yaml,yaml/responses.yaml")
+REPOSITORY_SUBFOLDER = os.getenv("REPOSITORY_SUBFOLDER")
+GITHUB_TOKEN = os.getenv("GITHUB_TOKEN")
+
+# TODO: validate
+BASE_URL = os.getenv("BASE_URL")
+
+# TODO: validate
+REPOSITORY_HANDLE = os.getenv("REPOSITORY_HANDLE")
+
+# TODO: validate
+INDEX_NAME = os.getenv("INDEX_NAME")
+
+# TODO: remove
+APIDOCS_BASE_URI = os.getenv("APIDOCS_BASE_URI")
+APIDOCS_BASE_PATH = os.getenv("APIDOCS_BASE_PATH")
+API_SPEC_FILES = os.getenv("API_SPEC_FILES")
+
 WORKDIR = os.getenv("WORKDIR", "/home/indexer")
 
 # Path to markdown files
 SOURCE_PATH = f'{WORKDIR}/gitcache'
+DOCS_INDEX_MAPPING = json.load(open("mappings/hugo.json", "rb"))
```

Comment:
It's potentially dangerous to not close file descriptors.
I think this should work too:
```
DOCS_INDEX_MAPPING = None
with open("mappings/hugo.json", "rb") as f:
    DOCS_INDEX_MAPPING = json.load(f)
```

Reply:
If that does not work, we could ignore it, because it will probably not make a difference atm.
Reference: https://stackoverflow.com/questions/7395542/is-explicitly-closing-files-important

Reply:
Done